2021/togt2022badder/journal/metadata.yaml

```diff
@@ -0,0 +1,132 @@
+# contributions" as name (with no address).
+affiliations:
+  - code: 1
+    name: University of Amsterdam
+    address: Amsterdam, the Netherlands
+  - code: 2
+    name: Equal contributions
+
+# List of keywords (adding the programming language might be a good idea)
+keywords:
+  rescience c, machine learning, deep learning, python, pytorch, nlp, bias,
+  seeds
+
+# Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
+# You can get a DOI for your code from Zenodo, or an SWH identifier from
+# Software Heritage.
+#   see https://guides.github.com/activities/citable-code/
+code:
+  - url: https://github.com/thesofakillers/badder-seeds
+  - swh: swh:1:dir:13ff45fd249e765a221d49f701c32d45b64ee675
+  - doi: 10.5281/zenodo.6480966
+
+# Information about the original article that has been replicated
+replication:
+  - cite:
+      "Maria Antoniak, David Mimno. Bad Seeds: Evaluating Lexical Methods for
+      Bias Measurement (ACL 2021)."
+  - bib: antoniak-mimno-2021-bad
+  - url: https://aclanthology.org/2021.acl-long.148/
+  - doi: 10.18653/v1/2021.acl-long.148
+
+# Don't forget to surround abstract with double quotes
+abstract:
+  "Combating bias in NLP requires bias measurement. Bias measurement is almost
+  always achieved by using lexicons of seed terms, i.e. sets of words specifying
+  stereotypes or dimensions of interest. This reproducibility study focuses on
+  the original authors' main claim that the rationale for the construction of
+  these lexicons needs thorough checking before usage, as the seeds used for
+  bias measurement can themselves exhibit biases. The study aims to evaluate the
+  reproducibility of the quantitative and qualitative results presented in the
+  paper and the conclusions drawn thereof. We reproduce most of the results
+  supporting the original authors' general claim: seed sets often suffer from
+  biases that affect their performance as a baseline for bias metrics.
+  Generally, our results mirror the original paper's. They are slightly
+  different on select occasions, but not in ways that undermine the paper's
+  general intent to show the fragility of seed sets."
+
+# Bibliography file (yours)
+bibliography: bibliography.bib
+
+# Type of the article
+# Type can be:
+#  * Editorial
+#  * Letter
+#  * Replication
+type: Replication
+
+# Scientific domain of the article (e.g. Computational Neuroscience)
+#  (one domain only & try to be not overly specific)
+domain: ML Reproducibility Challenge 2021
+
+# Coding language (main one only if several)
+language: Python
+
+# To be filled by the author(s) after acceptance
+# -----------------------------------------------------------------------------
+
+# For example, the URL of the GitHub issue where review actually occured
+review:
+  - url: https://openreview.net/forum?id=HcIxA3Mm2CF
+
+contributors:
+  - name: Koustuv Sinha
+    orcid:
+    role: editor
+  - name: Anonymous Reviewers
+    orcid:
+    role: reviewer
```

Comment:
Hi @thesofakillers , unfortunately the build script fails as the ReScience template expects two entries minimum for the role "reviewer". Please add a blank name block as we have provided in the template following this line.

```
- name:
    orcid:
    role: reviewer
```